wordpress-mobile/WordPress-Android
# Issue/6852 add username changer

### Fix
Add a username changer for social signup as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6852.

### Test
0. Clear app data.
1. Tap ***Sign Up*** button.
2. Tap ***Sign Up with Google*** button.
3. Choose Google account.
4. Notice ***Epilogue for Social*** screen.
5. Tap ***Username*** input field.
6. Notice ***Change Username*** screen with username suggestions.
7. Tap ***X*** action in toolbar.
8. Notice ***Username*** input field is unchanged.
9. Tap ***Username*** input field.
10. Tap any username from suggestions.
11. Notice username changes in header text.
12. Tap ***X*** action in toolbar.
13. Tap ***Discard*** action in dialog.
14. Notice ***Username*** input field is unchanged.
15. Tap ***Username*** input field.
16. Tap any username from suggestions.
17. Tap ***X*** action in toolbar.
18. Tap ***Cancel*** action in dialog.
19. Tap ***Save*** action in toolbar.
20. Notice ***Username*** input field has changed.
21. Tap ***Username*** input field.
22. Tap any username from suggestions except first suggestion.
23. Enter text to get new username suggestions.
24. Notice new username suggestions are displayed.
25. Notice selected username is first in suggestions.
